tests/test_wordpress_review_fixes.py

```
@@ -0,0 +1,145 @@
+        subprocess.run(
+            ["php", "-r", php],
+            cwd=ROOT,
+            text=True,
+            capture_output=True,
+            check=True,
+        ).stdout
+    )
+
+    assert results["valid_field_keys"] is True
+    assert results["valid_names"] is True
+    assert results["empty"] != True
```

Ruff (0.16.1)

[error] Avoid inequality comparisons to `True`; use `not results["empty"]:` for false checks

Replace with `not results["empty"]`

(E712)